backend/otr/src/main/java/fi/oph/otr/config/ConfigEnums.java

```diff
@@ -2,6 +2,7 @@
 
 public enum ConfigEnums {
   CALLER_ID("1.2.246.562.10.00000000001.otr"),
+  EMAIL_SENDER_NAME("Oikeustulkkirekisteri"),
```

Comment:
Tämä vähän niin ja näin onko fiksu lisäys tänne

Reply:
No, eiköhän se ole ok. Enemmän nyppii tuo että laitoin nämä copy pastena enumiksi :)

Reply:
Meinasit varmaan että mieluummin konffeista tulisi arvot? Sama mullakin kävi mielessä, mutta olkoon täällä nyt ja siirretään konffeihin myöhemmin.